tests/unit/configuration.py

```diff
@@ -103,10 +109,13 @@ def test_print_users(self):
         conf = self._build_test_config()
 
         f = io.StringIO()
-        with self.assertRaises(SystemExit) as cm, contextlib.redirect_stdout(f):
-            conf.print_users()
-        self.assertEqual(cm.exception.code, 0)
-        self.assertTrue("*  cli-dev" in f.getvalue())
+
+        try:
+            with contextlib.redirect_stdout(f):
+                conf.print_users()
+        except SystemExit as err:
+            assert err.code == 0
+            assert "*  cli-dev" in f.getvalue()
```

Comment:
https://docs.pytest.org/en/latest/how-to/assert.html#assertions-about-expected-exceptions

Might be better to use `pytest.raises` context manager here.

Reply:
Think this should be ok for now since it is actually catching the error and asserting properly